<launch>
    <!-- RVIZ -->
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find aruco_mapping)/launch/aruco_config.rviz" />

    <!--   usb_cam node -->
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen">
        <param name="video_device" value="/dev/video1" />
        <param name="image_width" value="1280" />
        <param name="image_height" value="720" />
        <param name="pixel_format" value="yuyv" />
        <param name="camera_frame_id" value="usb_cam" />
        <param name="io_method" value="mmap"/>
    </node>

    <!-- <include file="$(find lslidar_c16_decoder)/launch/lslidar_c16.launch" /> -->

    <!-- <param name="/use_sim_time" value="true"/> -->
    <!-- ArUco mapping -->
    <node pkg="aruco_mapping" type="aruco_mapping" name="aruco_mapping" output="screen">
        <remap from="/image_raw" to="/usb_cam/image_raw"/>

        <param name="calibration_file" type="string" value="$(find calibration)/params/calibrated_camera_1121.ini"/>
        <param name="num_of_markers" type="int" value="2" />
        <param name="marker_size" type="double" value="0.135"/>
        <param name="space_type" type="string" value="plane" />
        <param name="roi_allowed" type="bool" value="false" />
    </node>


    <!-- <rosparam command="load" file="$(find calibration)/params/lidar_camera_calibration.yaml" />
    <node pkg="lidar_camera_calibration" type="find_transform" name="find_transform" output="screen">
    </node> -->
</launch>